app/src/main/res/layout/activity_create_event.xml

```diff
@@ -1,182 +1,191 @@
+            app:layout_constraintStart_toStartOf="parent" />
+
+        <EditText
+            android:id="@+id/etMaxPlayers"
+            android:layout_width="165dp"
+            android:layout_height="wrap_content"
+            android:hint="@string/max_players"
+            android:inputType="number"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toEndOf="@+id/etMinPlayers"
+            app:layout_constraintTop_toTopOf="@+id/etMinPlayers" />
+
+        <Spinner
+            android:id="@+id/spSkillLevel"
+            android:layout_width="@dimen/spinnerDim"
+            android:layout_height="@dimen/spinnerDim"
+            android:background="@android:drawable/btn_dropdown"
+            android:spinnerMode="dropdown"
+            app:layout_constraintEnd_toStartOf="@id/tvAllowPlusOnes"
+            app:layout_constraintStart_toEndOf="@+id/tvSkillLevel"
+            app:layout_constraintTop_toTopOf="@+id/tvSkillLevel" />
+
+
+        <Spinner
+            android:id="@+id/spAllowPlusOnes"
+            android:layout_width="@dimen/spinnerDim"
+            android:layout_height="@dimen/spinnerDim"
+            android:background="@android:drawable/btn_dropdown"
+            android:spinnerMode="dropdown"
+            app:layout_constraintEnd_toStartOf="@id/tvAllowSpectators"
+            app:layout_constraintStart_toEndOf="@+id/tvAllowPlusOnes"
+            app:layout_constraintTop_toTopOf="@+id/tvAllowPlusOnes" />
+
+        <Spinner
+            android:id="@+id/spAllowSpectators"
+            android:layout_width="@dimen/spinnerDim"
+            android:layout_height="@dimen/spinnerDim"
+            android:background="@android:drawable/btn_dropdown"
+            android:spinnerMode="dropdown"
+
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintHorizontal_bias="0.0"
+            app:layout_constraintStart_toEndOf="@+id/tvAllowSpectators"
+            app:layout_constraintTop_toTopOf="@+id/tvAllowSpectators" />
+
+        <EditText
+            android:id="@+id/etGym"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:hint="@string/gym_pointer"
+            android:inputType="textPostalAddress"
+            app:layout_constraintBottom_toTopOf="@+id/btCreateEvent"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintHorizontal_bias="0.0"
+            app:layout_constraintStart_toStartOf="parent" />
+
+        <EditText
+            android:id="@+id/etTeamRotation"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:hint="@string/rotation_rule"
+            android:inputType="textMultiLine"
+            android:layout_marginBottom="16dp"
+            app:layout_constraintBottom_toTopOf="@+id/etGym"
+            tools:layout_editor_absoluteX="36dp" />
```

Comment:
Try to avoid setting absolute values like this.  Lay things out relative to eachother and use margins to space them.